meson.build

```diff
@@ -6,6 +6,11 @@ project('piper',
 # Dependencies
 dependency('python3', required: true)
 dependency('pygobject-3.0', required: true)
+ratbagd = find_program('ratbagd', required: false)
+
+if not ratbagd.found()
```

Comment:
Doesn't this do exactly the same as setting `required: true` in the `find_program` directive?